packages/inter-protocol/package.json

```diff
@@ -10,8 +10,8 @@
   "scripts": {
     "build": "yarn build:bundles",
     "build:bundles": "node ./scripts/build-bundles.js",
-    "test": "ava",
-    "test:c8": "c8 $C8_OPTIONS ava --config=ava-nesm.config.js",
+    "test": "rm -f bundles/*.lock && ava",
```

Comment:
Shouldn't the bundle cache figure out when to rebuild?

Reply:
This is just cleaning up stray lock files (from bundling that was badly interrupted), not affecting future rebuilds.

Reply:
I don't follow. What's the consequence of leaving this out? Why is that important only in this package and nowhere else?

Reply:
thanks, I'd prefer to leave this change out and if the problem it's solving recurs then to tackle it in a way that's more comprehensive and easier to maintain